pageserver/src/config.rs

```diff
@@ -144,6 +144,7 @@ pub struct S3Config {
     pub access_key_id: Option<String>,
     /// "Password" to use when connecting to bucket.
     pub secret_access_key: Option<String>,
+    pub endpoint: Option<String>,
```

Comment:
What does it mean to have None endpoint?

Reply:
Right, documented it.
This is a way to use non-AWS S3 set-ups like our mocking.